ipn/ipnlocal/c2n_pprof.go

```diff
@@ -7,11 +7,17 @@ package ipnlocal
 
 import (
 	"net/http"
+	"runtime"
 	"runtime/pprof"
+	"strconv"
 )
 
 func init() {
 	c2nLogHeap = func(w http.ResponseWriter, r *http.Request) {
+		// Support same optional gc parameter as net/http/pprof:
```

Comment:
I thought net/http/pprof uses a "/gc" URL endpoint, rather than a URL query parameter?

Reply:
Not that I see. This code is straight from net/http/pprof. And docs say:

> heap: A sampling of memory allocations of live objects. You can specify the gc GET parameter to run GC before taking the heap sample.